src/webapp/components/table/performance-overview-table/PerformanceOverviewTable.tsx

```diff
@@ -0,0 +1,277 @@
+import React, { useEffect, useMemo, useState } from "react";
+import _ from "lodash";
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableHead,
+    TableRow,
+    TableContainer,
+} from "@material-ui/core";
+import styled from "styled-components";
+import { SearchInput } from "../../search-input/SearchInput";
+import { Selector } from "../../selector/Selector";
+import { Maybe } from "../../../../utils/ts-utils";
+import i18n from "../../../../utils/i18n";
+
+export type TableColumn = {
+    value: string;
+    label: string;
+    dark?: boolean;
+};
+
+interface PerformanceOverviewTableProps {
+    rows: {
+        [key: TableColumn["value"]]: string;
+    }[];
+}
+
+export const PerformanceOverviewTable: React.FC<PerformanceOverviewTableProps> = React.memo(
```

Comment:
Would it be better to have a more generic name for the table like so that it can be used in other places if required?

Reply:
I'm very (but very) bad with the names but I like that one :)